etc/cp-schema-registry/values.yaml

```diff
@@ -19,7 +17,6 @@ resources:
 ingress:
   enabled: true
   annotations:
-    kubernetes.io/ingress.class: nginx
```

Comment:
Do we need to replace this with `ingressClassName: "nginx"`?

Reply:
Yes but it is in the latest version of the schema registry helm chart so there shouldn't be an issue.

Reply:
Oh okay thank you! Is it `0.3.2`?

Reply:
Yes!